src/Repository/Schedule/ScheduleRepository.php

```diff
@@ -0,0 +1,67 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Repository\Schedule;
+
+use App\Dto\ScheduleRequest;
+use App\Entity\Schedule;
+use Doctrine\ORM\EntityManagerInterface;
+use Ramsey\Uuid\Uuid;
+use Ramsey\Uuid\UuidInterface;
+
+final class ScheduleRepository implements ScheduleRepositoryInterface
+{
+    private $entityManager;
+    private $repository;
+
+    public function __construct(EntityManagerInterface $entityManager)
+    {
+        $this->entityManager = $entityManager;
+        $this->repository = $entityManager->getRepository(Schedule::class);
+    }
+
+    public function find(string $id): ?Schedule
+    {
+        return $this->repository->find($id);
+    }
+
+    public function findAll()
```

Comment:
Add a return type. Have a look if you have other findAll method in your code without return type